packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts

```diff
@@ -0,0 +1,519 @@
+import { ITriggerFunctions } from 'n8n-core';
+import {
+	IDataObject,
+	INodeType,
+	INodeTypeDescription,
+	ITriggerResponse,
+	NodeOperationError,
+} from 'n8n-workflow';
+
+import { CronJob } from 'cron';
+import { ICronExpression } from './CronInterface';
+import moment from 'moment';
+
+export class ScheduleTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Schedule Trigger',
+		name: 'scheduleTrigger',
+		icon: 'file:schedule.svg',
+		group: ['trigger', 'schedule'],
+		version: 1,
+		description: 'Triggers the workflow on a given schedule',
+		eventTriggerDescription: '',
+		activationMessage:
+			'Your schedule trigger will now trigger executions on the schedule you have defined.',
+		defaults: {
+			name: 'Schedule Trigger',
+			color: '#00FF00',
+		},
+		// eslint-disable-next-line n8n-nodes-base/node-class-description-inputs-wrong-regular-node
+		inputs: [],
+		outputs: ['main'],
+		properties: [
+			{
+				displayName:
+					'This workflow will run on the schedule you define here once you <a data-key="activate">activate</a> it.<br><br>For testing, you can also trigger it manually: by going back to the canvas and clicking ‘execute workflow’',
+				name: 'notice',
+				type: 'notice',
+				default: '',
+			},
+			{
+				displayName: 'Trigger Rules',
+				name: 'rule',
+				placeholder: 'Add Rule',
+				type: 'fixedCollection',
+				typeOptions: {
+					multipleValues: true,
+				},
+				default: {
+					interval: [
+						{
+							field: 'days',
+						},
+					],
+				},
+				options: [
+					{
+						name: 'interval',
+						displayName: 'Trigger Interval',
+						values: [
+							{
+								displayName: 'Trigger Interval',
+								name: 'field',
+								type: 'options',
+								default: 'days',
+								options: [
```

Comment:
I know this list is ordered alphabetically but isn't it best if we order it from smallest to largest?